documentation/cloud-manager-guide/environments/environment-details/environment-overview.md

```diff
@@ -0,0 +1,145 @@
+---
+keywords:
+title: Environment Overview
+description: Components and navigation for an environment's detailed view
+---
+
+# Environment Overview
+
+Each environment has a detailed view where you can see further information about the environment, monitor its status, and perform various operations on the environment. This guide outlines the features of the *Overview* screen and how to navigate the detailed view of an environment.
```

Comment:
"This guide outlines the detailed view of an environment as seen in the environment *Overview* screen." Or something to make it clear this is not the other Overview area.